catalog/ui/catalog-ui-search/src/main/webapp/component/golden-layout/golden-layout.view.js

```diff
@@ -185,7 +213,7 @@ module.exports = Marionette.LayoutView.extend({
             currentConfig = defaultGoldenLayoutContent;
         }
         _merge(currentConfig, getGoldenLayoutSettings());
-        return currentConfig;
+        return sanitizeTree(currentConfig);
```

Comment:
We should be doing this in the handleGoldenLayoutStateChange where we put it in user prefs.